client/packages/plugins/backendCommon/utils.ts

```diff
@@ -0,0 +1,55 @@
+const wrapSql = (fields: string[], sqlStatement: string) => {
```

Comment:
I created these helpers in civ plugins, copied over here now that we are adding new interface, will update in civ plugin

Reply:
Nice to have these in a common place! Would be good to see unit tests around them though 😉

Reply:
Agree, added here: https://github.com/msupply-foundation/open-msupply/issues/2951#issuecomment-2650673745